src/handler.test.js

```diff
@@ -381,3 +382,25 @@ test("Response with malformed JSON returns an error", async () => {
     expect(result.body).toContain('"message":"An error has occurred."');
     expect(result.body).toContain('"timestamp":');
 });
+
+test("the numberOfMatches value is returned", async () => {
+    const event = {
+        queryStringParameters: {
+            search: "physics",
+        },
+    };
+    const searchResults = {
+        numberOfMatches: 3,
+        results: [],
+    };
+
+    const expectedResult = {
+        numberOfMatches: 3,
+        results: [],
+    };
+
+    fetch.mockResponse(JSON.stringify(searchResults), { status: 200 });
+    const result = await courses(event);
+    expect(result.statusCode).toBe(200);
+    expect(result.body).toEqual(JSON.stringify(expectedResult));
+});
```

Comment:
This is a way that we could make this test only check the `numberOfMatches` :slightly_smiling_face: 

```suggestion

    fetch.mockResponse(JSON.stringify(searchResults), { status: 200 });
    const result = await courses(event);
    expect(result.statusCode).toBe(200);
    expect(JSON.parse(result.body).numberOfMatches).toEqual(3);
```